src/Common/src/Common/HealthChecks/HealthCheckResult.cs

```diff
@@ -31,5 +36,6 @@ public sealed class HealthCheckResult
     /// <remarks>
     /// For parity with Spring Boot, repeat the status (in all-caps snake case) here.
     /// </remarks>
+    [JsonIgnoreEmptyCollection]
```

Comment:
```suggestion
    [JsonIgnoreEmptyCollection]
```

Reply:
This change will be in the next PR, which will also remove all of those repeated statuses from health contributors